src/Tests/SNIPPETS.cs

```diff
@@ -77,6 +77,21 @@ @ EntryPoint (  )
     }
 ";
 
+        public static string Op6b_Op6a =
+@"
+    /// # Summary
+    ///     This to show that the order of snippet operations is preserved:
+    operation Op6b() : Unit
+    {
+        HelloQ();
+    }
+
+    operation Op6a() : Unit
+    {
+        Op6b();
```

Comment:
NIT: Since Op6a() calls Op6b(), then the assert compile should see Op6b first and then Op6a? It might help here to add comments on what the intention of the test is.

Reply:
Thanks! I've clarified this a bit.